pysmarthashtag/api/authentication.py

```
@@ -515,8 +657,15 @@ async def async_auth_flow(self, request: Request) -> AsyncGenerator[Request, Res
     try:
-        retry_after = next(iter([int(i) for i in response.json().get("message", "") if i.isdigit()]))
+        retry_after_header = response.headers.get("Retry-After")
+        if retry_after_header and retry_after_header.isdigit():
+            retry_after = int(retry_after_header)
+        else:
+            match = re.search(r"\d+", response.json().get("message", ""))
+            if match:
+                retry_after = int(match.group())
     except Exception:
-        retry_after = 2
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)